translations/fasteners_pt-br.ts

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_BR" sourcelanguage="en_US">
+<context>
+    <name>InitGui</name>
+    <message>
+        <location filename="a2plib.py" line="1130"/>
```

Comment:
a2plib.py ?

Reply:
I didn't use an automated script to create the .ts file. So I copied the .ts file from the A2Plus bench and edited it manually and forgot to change the file location reference. The translation worked anyway. In the future I will add a script to generate the .ts files automatically and this issue will no longer occur.

Reply:
The location filename has been fixed.